harness/benchmarks/gd/Simple.gd

```diff
@@ -1,46 +1,51 @@
 func benchmark_simple_add():
```

Comment:
Shouldn't we now use strong typing in gdscript as well? IIRC the perf benefits of typed gdscript have been backported to 3.2 and will be available with the next patch release. IMO it would be more fair for gdscript to benefit from that in the benchmark.
Best would be to have two gdscript benchmarks: one typed and one not.

Reply:
@CedNaru ?

Reply:
Done. But there is no difference between Typed and Dynamic yet. Those changes are only for Godot 4.0

![image](https://user-images.githubusercontent.com/31470327/101607171-8c1e2c80-3a04-11eb-906d-d39aeaeb6d27.png)

Reply:
Cool thx. IIRC they are in the process of being backported to 3.x (but i can't find the PR anymore). But maybe I'm just wrong. Anyways good to have it.